components/collection/drop/HolderOfGenerative.vue

```diff
@@ -293,6 +295,11 @@ const mintNft = async () => {
 
 watch(status, (curStatus) => {
   if (curStatus === TransactionStatus.Block) {
+    if (isTransactionError.value) {
```

Comment:
add this to paid generative also?

Reply:
we can do follow up if so

Reply:
> add this to paid generative also?

Paid generative has handle tx error during the auto-teleport